pkg/operatorstatus/operatorstatus.go

```diff
@@ -10,123 +10,133 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/client-go/util/retry"
 
-	osapi "github.com/openshift/cluster-version-operator/pkg/apis/operatorstatus.openshift.io/v1"
+	configv1 "github.com/openshift/api/config/v1"
 	"github.com/operator-framework/operator-sdk/pkg/sdk"
 
 	"github.com/openshift/cluster-samples-operator/pkg/apis/samplesoperator/v1alpha1"
 )
 
-// CVOOperatorStatusHandler allows for wrappering sdk access to osapi.ClusterOperator
+// CVOOperatorStatusHandler allows for wrappering sdk access to configv1.ClusterOperator
 type CVOOperatorStatusHandler struct {
 	SDKwrapper OperatorStatusSDKWrapper
-	Namespace  string
 }
 
 // NewCVOOperatorStatusHandler is the default initializer function for CVOOperatorStatusHandler
-func NewCVOOperatorStatusHandler(namespace string) *CVOOperatorStatusHandler {
-	return &CVOOperatorStatusHandler{SDKwrapper: &defaultOperatorStatusSDKWrapper{}, Namespace: namespace}
+func NewCVOOperatorStatusHandler(clientGoWrapper *OperatorStatusSDKWrapper) *CVOOperatorStatusHandler {
+	handler := CVOOperatorStatusHandler{}
+	if clientGoWrapper != nil {
+		handler.SDKwrapper = *clientGoWrapper
+	} else {
+		handler.SDKwrapper = &defaultOperatorStatusSDKWrapper{}
```

Comment:
since this doesn't even work, when are we ever using it? can we delete this codepath?

Reply:
I kept it around with the reconsidering the sdk again when we upgraded the sdk in mind, and it also highlighted where I made the switch

but sure I can delete the codepath ... from the pointers from the network-edge team you got, the sdk api is different enough that keeping the old thing is less useful than I had anticipated

will delete